assets/css/about.css

```diff
@@ -137,3 +137,185 @@ footer img {
   opacity: 0.6;
   z-index: 1;
 }
+
+.footer {
+  flex-direction: column;
+}
+
+/* media queries */
+
+@media screen and (min-width: 768px) {
+  .about h1 {
+    font-size: 30px;
+  }
+
+  .about .opener {
+    font-size: 17px;
+  }
+
+  .overview {
+    font-size: 17px;
+    width: 500px;
+    padding: 5px;
+  }
+
+  .heading {
+    font-size: 25px;
+  }
+
+  .logo-container img {
+    width: 50%;
+    height: 40%;
+    margin: -50px auto -40px auto;
+  }
+
+  .past-conf-images {
+    display: flex;
+    justify-content: center;
+    gap: 40px;
+    margin: 0 auto;
+  }
+
+  .past-conf-images p {
+    width: 300px;
+    margin-bottom: 30px;
+  }
+
+  .past-conf-images p:first-child {
+    margin-right: -20px;
+  }
+
+  .footer {
+    justify-content: space-around;
+  }
+
+  .footer img {
+    width: 80px;
+    height: 80px;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .about {
+    height: 95vh;
+  }
+
+  .header .fa-bars {
+    display: none;
+  }
+
+  .header-top {
+    display: flex;
+    padding: 0 10px;
+  }
+
+  .header-top li a,
+  .header-top li a i {
+    color: #fff;
+  }
+
+  .header-bottom {
+    display: flex;
+    padding: 0 30px;
+  }
+
+  .header__navLink {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    position: unset;
+    height: 60px;
+    gap: 0;
+    font-weight: var(--fw-400);
+    background: none;
+    padding-top: 0;
+    font-size: 20px;
+    left: unset;
+    right: unset;
+    top: unset;
+    transition: none;
+  }
+
+  .navbar .header__navLink .nav-item {
+    color: var(--sl-black);
+  }
+
+  .nav-button {
+    display: inline-block;
+    padding: 5px;
+  }
+
+  .header__navLink .nav-button a {
+    color: var(--sl-orange);
+  }
+
+  .heading {
+    padding-top: 3rem;
+  }
+
+  .about h1 {
+    font-size: 50px;
+  }
+
+  .about .opener {
+    padding-top: 55px;
+    font-size: 25px;
+  }
+
+  .overview {
+    font-size: 25px;
+    width: 900px;
+    padding: 10px;
+  }
+
+  .contact-info {
+    font-size: 20px;
+  }
+
+  .logo-details p {
+    font-size: 20px;
+    margin-top: -20px;
+  }
+
+  .logo-container img {
+    width: 30%;
+    height: 40%;
+    margin: -120px auto -60px auto;
+  }
+
+  .past-conf-images p {
+    width: 400px;
+    margin-bottom: 30px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .header {
+    display: block;
+  }
+
+  .heading {
+    padding: 2rem;
+    font-size: 30px;
+  }
+
+  .header .fa-bars {
+    display: none;
+  }
+
+  .header-bottom,
+  .header-top {
+    display: flex;
+    padding: 0 130px;
+    color: var(--sl-black);
+  }
+
+  .header-bottom {
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .footer {
+    padding: 0 130px;
+  }
```

Comment:
- [ ] Great work @joseph07-drack , just as the previous reviewer has mentioned kindly change the background color of your footer to match the given website design

Reply:
All good now 👍🏽

Reply:
- [ ] [OPTIONAL] Kindly note that the homepage for the mobile is not supposed to have a footer since you added the more button. The about page is supposed to have the footer exactly as it was implemented on the homepage and not how it was implemented on the about page.

Mobile(Home Page)   |  Mobile(About Page)
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-13 13-06-21](https://user-images.githubusercontent.com/58692590/163180097-57ac5f26-5396-402e-b238-49a7d02b8035.png) | ![Screenshot from 2022-04-13 13-06-51](https://user-images.githubusercontent.com/58692590/163180125-d8ae85e5-bc38-4651-9ac8-483d6fa75eee.png)